db/item_group_db.yml

```diff
@@ -0,0 +1,51 @@
+# This file is a part of rAthena.
+#   Copyright(C) 2021 rAthena Development Team
+#   https://rathena.org - https://github.com/rathena
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Item Group Database
+###########################################################################
+#
+# Item Group Settings
+#
+###########################################################################
+# - Group                      Group name of the item. 'IG_' is appended to the name when the file is read.
+#   Contain:
+#     - SubGroup:              SubGroup number associated with the list of item. 0 is exclusive to 'Must' SubGroup, a SubGroup which doesn't use Rate. 
+#                              Anything above 0 is for 'Random' SubGroup. 'Random' SubGroup requires Rate. Read doc/item_group.txt for more informations.
+#       List:                  List of item(s) for the SubGroup.
+#        - Item                Available item that will be obtained from this item group. Requires the AegisName of the item.
+#          Rate                Probability to get the item. Not a percentage value!
+#          Amount              Amount of item that will be obtained (Default: 1).
+#          Duration            Makes the item a rental item, which will be expire in the given amount of minutes. Not intended for use with stackable items (Default: 0).
+#          Announced           If player obtained this item, it will be broadcast to the server (true/false. Default: false).
+#          GUID                Makes the given item(s) with Unique ID (true/false. Default: false).
+#          Named               Inscribes the item with the obtainer's name (true/false. Default: false).
+#          Bound               Binds the obtained item. CHAR, ACCOUNT, GUILD, PARTY or NONE (Default: NONE).
+#       Clear                  Remove the given SubGroup (Optional).
```

Comment:
```suggestion
# - Group                      Group name of the item.
#   Contain:
#     - SubGroup:              SubGroup number associated with the list of item. See doc/item_group.txt for more information.
#       List:                  List of item(s) for the SubGroup.
#        - Item                AegisName of item that will be obtained from this item group.
#          Rate                Probability to get the item. See doc/item_group.txt for more information. (Default: 0)
#          Amount              Amount of item that will be obtained. (Default: 1)
#          Duration            Makes the item a rental item which will expire within the given amount in minutes. Not intended for use with stackable items. (Default: 0)
#          Announced           If player obtains this item it will be broadcasted to the server. (Default: false)
#          GUID                Makes the given item(s) have a Unique ID. (Default: false)
#          Named               Inscribes the item with the obtainer's name. (Default: false)
#          Bound               Binds the obtained item. (Default: None)
#       Clear                  Remove the given SubGroup. (Optional)
```